src/composition.jl

```diff
@@ -42,6 +41,15 @@ for (f, _f, g) in ((:issymmetric, :_issymmetric, :transpose),
     end
 end
 
+# A * B * A and A * B * A' are positive definite if (sufficient condition) A & B are positive definite
+LinearAlgebra.isposdef(A::CompositeMap) = _isposdef(A.maps)
```

Comment:
With the latest #154, this should be
```suggestion
_isposdef(A::CompositeMap) = _isposdef(A.maps)
```
if I'm not mistaken. Since we are very conservative about checking positive definiteness, I think we should give `CompositeMaps` a chance, in case somebody bothered to manually set the `isposdef` flag `true` for the factors. I mean I'm in favor of keeping this sufficient criterion instead of just defaulting to `false` for any `CompositeMap`.

Reply:
I agree that we should see if the sufficient condition holds.
I tried to sync my branch with main branch but somehow my attempt to resolve the conflict didn't work.
I hope I didn't mess it up...

Reply:
Sorry, I confused myself. This was correct. `_isposdef(C::CompositeMap)` will fall back to `isposdef(C)` which will then enter the recursion here.